tests/integration/defs/perf/test_perf.py

```diff
@@ -1288,6 +1401,10 @@ def get_perf_result(self, outputs: Dict[int, str]) -> float:
         if len(metric_values) == 0:
             if self._build_script == "trtllm-build" and metric.metric_type == PerfMetricType.ENGINE_SIZE:
                 metric_values = [0.0]
+            elif self._build_script == "trtllm-bench" and self._config.num_gpus > 1 and metric.metric_type == PerfMetricType.BUILD_TIME:
+                print_info("skip building process for multi-gpu test"
```

Comment:
Please correct me if my understanding is wrong - this only skip the metric, but not the build process, right?
```suggestion
                print_info("WAR: skip build time metric for multi-gpu test"
```

Reply:
yes, thanks for suggestion